constrained-voucher.mkd

```diff
@@ -1343,3 +1344,56 @@ INSERT_FIG_FROM_FILE examples/voucher-from-MASA.txt END
 The representiation of signed_voucher in CBOR diagnostic format is:
 
 INSERT_FIG_FROM_FILE examples/voucher-from-MASAdiag.txt END
+
+
+# Pledge Device Class Profiles {#appendix-pledge-profiles}
+
+This specification allows implementers to select between various functional options for the Pledge, 
+yielding different code size footprints and different requirements on Pledge hardware.
+Thus for each product an optimal trade-off between functionality, development/maintenance cost and hardware cost can be made.
+
+This appendix illustrates different selection outcomes by means of defining different example "profiles" of constrained Pledges. In the following
+subsections, these profiles are defined and a comparison is provided.
+
+## Minimal Pledge
+The Minimal Pledge profile (Min) aims to reduce code size and hardware cost to a minimum. This comes with some severe functional restrictions, in particular:
+
+* No support for EST re-enrollment: whenever this would be needed, a factory reset followed by a new bootstrap process is required.
+* No support for change of Registrar: for this case, a factory reset followed by a new bootstrap process is required.
+
```

Comment:
```suggestion

This profile would be appropriate for single-use devices which must be replaced rather than re-deployed. 
That might  include medical devices, but also sensors used during construction, such as concrete temperature  sensors.
```